ts/src/btcalpha.ts

```diff
@@ -949,27 +949,20 @@ export default class btcalpha extends Exchange {
         return { 'url': url, 'method': method, 'body': body, 'headers': headers };
     }
 
-    handleErrors (code: int, reason: string, url: string, method: string, headers: Dict, body: string, response, requestHeaders, requestBody) {
+    handleErrors (httpCode: int, reason: string, url: string, method: string, headers: Dict, body: string, response, requestHeaders, requestBody) {
```

Comment:
@egorbulat98 thanks for your contribution, can you please restore this parameter's name? No need to change it here